pkg/cmd/pulumi/convert-trace.go

```diff
@@ -425,6 +425,11 @@ func (s *otelSpan) InstrumentationLibrary() instrumentation.Library {
 	return instrumentation.Library{Name: "pulumi-convert"}
 }
 
+// InstrumentationScope returns the scope of span.
+func (s *otelSpan) InstrumentationScope() instrumentation.Scope {
+	return instrumentation.Scope{}
```

Comment:
Not sure we have the info avaialbe to fill in here, I think this would be the go module from where the span originated.